app/assets/javascripts/dradis/plugins/csv/upload.js

```diff
@@ -6,8 +6,9 @@ window.addEventListener('job-done', function(e){
     if (uploader.value === 'Dradis::Plugins::CSV') {
       var path = window.location.pathname;
       var project_path = path.split('/').slice(0, -1).join('/');
+      var attachment = $('#filename').val();
```

Comment:
Would there be conflicts with https://github.com/dradis/dradis-ce/blob/develop/app/assets/javascripts/tylium/modules/uploads.js.coffee#L31? Both have the id `filename`.

What about just grabbing the filename like: `$('#filename').text()`?